### ISSUE TITLE
Make Eldev documentation available as an Info manual when installed through MELPA

### ISSUE THREAD

**Issue by suhail-singh** [emacs-eldev/eldev#110]
While the Eldev website has excellent documentation, at times I do wish it was
easier to peruse it offline and especially within Emacs as an Info manual (with
keyboard navigation to navigate sections etc).

If the MELPA package contained the documentation in Info format, it would help
matters.

**Comment by doublep** [comment]
I certainly don't plan to maintain two sets of documentation, but if you can find an automated converter from Asciidoc, I could try to autoconvert the manual into `.info` format.

**Comment by doublep** [comment]
> Having not tried this particular path myself (though I have used org-pandoc-import as well as ox-texinfo in the past), I am not sure how usable the generated Info manual would be

M-m, can you at least try that? I don't use Org myself, so I don't want to spend time investigating this if it's not even certain the end result is going to be useful.

It would be great if you could provide some short (Lisp?) code to simply do it. I can then convert it to a GitHub workflow (given that the package needs to be published on MELPA, I doubt I could include any forms to "run before packaging", I can only package files already in the source tree).

**Comment by doublep** [comment]
> unless you're willing to maintain the documentation in Org format 

No, sorry. I don't want to change from Asciidoc. I'll keep the issue open in case someone finds a way to autoconvert that to `.info` somehow.

**Comment by okomestudio** [comment]
I was looking to see if the Info version of manual existed for Eldev and came across this ticket. I experimented with creating the .info file, similarly to https://github.com/emacs-eldev/eldev/issues/110#issuecomment-2547325447, and the conversion from Org to Info seems to work so long as the hyperlinks to marked objects within the same Org document is in the correct format.

That conversion process is hard-coded in the PR that I just created. 

The Info document is viewable with Emacs, but I'm not sure the generated Info doc is sufficiently well-formatted for immediate use. The original version contains the table of contents, but that looks redundant when rendered. So I'm not sure how that kind of thing should be dealt with. 

https://github.com/emacs-eldev/eldev/pull/112 is more a proof-of-concept PR, so I'm not necessarily requesting it to be merged as is. 

I hope this moves things closer for Eldev to actually include the Info version of documentation. The asciidoc version is pretty nice, but not needing to switch between Emacs and the web browser frequently is also nice.
